### ISSUE TITLE
False positive in empty_enum_arguments rule

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1549]
```swift
public var title: String? {
    switch (name, city) {
    case let (name?, city?):
        return "\(name), \(city)"
    case let (name?, _):
        return name
    case let (_, city?):
        return city
    case (_, _): // this triggers
        return nil
    }
}
```
